File: Leetcode Challenge/March/Java/Two City Scheduling.java
class Solution {
    public int twoCitySchedCost(int[][] costs) {
        Arrays.sort(costs, (a, b) -> {
            return (a[0] - a[1]) - (b[0] - b[1]);
        });
        
        int price = 0;
        for(int i = 0; i < costs.length / 2; i++){
            price += costs[i][0];
        }
        for(int i = costs.length / 2; i < costs.length; i++){
            price += costs[i][1];
        }
        return price;
    }
}

File: Leetcode Challenge/March/Python/Two City Scheduling.py
class Solution:
    def twoCitySchedCost(self, costs: List[List[int]]) -> int:
        
        # costs.sort(key=lambda cost: cost[0] - cost[1])
        costs.sort(key=lambda cost: cost[0] - cost[1])
        print(costs)
        result=0
        
        for i in range(len(costs)//2):
            result+=costs[i][0]
        
        for j in range(len(costs)-1, (len(costs)//2)-1, -1):
    
            result+=costs[j][1]
        return result
